ddtrace/contrib/kafka/patch.py

```diff
@@ -27,9 +27,11 @@
 
 _Producer = confluent_kafka.Producer
 _Consumer = confluent_kafka.Consumer
-_SerializingProducer = confluent_kafka.SerializingProducer if hasattr(confluent_kafka, "SerializingProducer") else None
+_has_serializing_producer = hasattr(confluent_kafka, "SerializingProducer")
+_has_deserializing_consumer = hasattr(confluent_kafka, "DeserializingConsumer")
+_SerializingProducer = confluent_kafka.SerializingProducer if _has_serializing_producer else confluent_kafka.Producer
 _DeserializingConsumer = (
-    confluent_kafka.DeserializingConsumer if hasattr(confluent_kafka, "DeserializingConsumer") else None
+    confluent_kafka.DeserializingConsumer if _has_deserializing_consumer else confluent_kafka.Consumer
```

Comment:
Can't subclass None, so defaulting to regular Consumer in order to subclass in the Tracing version.